app/db/user_models.py

```diff
@@ -142,3 +144,16 @@ class UserAnswer(BaseTable):
     result = relationship('Result')
     question = relationship('Question')
     answer = relationship('Answer')
+
+class NotificationStatus(Enum):
+    UNREAD = 'unread'
+    READ = 'read'
+
+class Notification(BaseTable):
+    __tablename__ = 'notifications'
+
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
```

Comment:
try to add created_at and updated_at to the BaseTable and use for every model, this can be very helpful